notebooks/121-convert-to-openvino/121-convert-to-openvino.ipynb

```diff
@@ -0,0 +1,1648 @@
+    "\n",
+    "if ONNX_CV_MODEL_PATH.exists():\n",
+    "    print(f\"ONNX model {ONNX_CV_MODEL_PATH} already exists.\")\n",
+    "else:\n",
+    "    with warnings.catch_warnings():\n",
+    "        warnings.filterwarnings(\"ignore\")\n",
+    "        torch.onnx.export(\n",
+    "            model=pytorch_model,\n",
+    "            args=torch.randn(1, 3, 780, 520),\n",
+    "            f=ONNX_CV_MODEL_PATH\n",
+    "        )\n",
+    "    print(f\"ONNX model exported to {ONNX_CV_MODEL_PATH}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Basic conversion\n",
+    "\n",
+    "To convert a model to OpenVINO IR, use the following command:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
+      "To disable this warning, you can either:\n",
+      "\t- Avoid using `tokenizers` before the fork if possible\n",
+      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
+      "[ INFO ] The model was converted to IR v11, the latest model format that corresponds to the source DL framework input/output format. While IR v11 is backwards compatible with OpenVINO Inference Engine API v1.0, please use API v2.0 (as of 2022.1) to take advantage of the latest improvements in IR v11.\n",
+      "Find more information about API v2.0 and IR v11 at https://docs.openvino.ai/2023.0/openvino_2_0_transition_guide.html\n",
+      "[ SUCCESS ] Generated IR version 11 model.\n",
+      "[ SUCCESS ] XML file: /home/asuvorov/projects/openvino/openvino_notebooks/notebooks/121-convert-to-openvino/model/distilbert.xml\n",
+      "[ SUCCESS ] BIN file: /home/asuvorov/projects/openvino/openvino_notebooks/notebooks/121-convert-to-openvino/model/distilbert.bin\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Model Optimizer CLI\n",
+    "\n",
+    "! mo --input_model model/distilbert.onnx --output_dir model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
+      "To disable this warning, you can either:\n",
+      "\t- Avoid using `tokenizers` before the fork if possible\n",
+      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Python conversion API\n",
+    "from openvino.tools.mo import convert_model\n",
+    "\n",
+    "ov_model = convert_model(ONNX_NLP_MODEL_PATH)"
```

Comment:
no message here, previous shows  SUCCESS  Generated IR version 11 model. In this one I have no indication that conversion was successful 
maybe add a note saying that to execute the cell, change ONNX_CV_MODEL_PATH to your model path? I'm not sure how this is supposed to work

Reply:
discussed